exist-docker/src/main/resources-filtered/Dockerfile-DEBUG

```diff
@@ -21,7 +21,7 @@
 #
 
 # Use JDK 17 in Debian Bullseye (as our production image gcr.io/distroless/java:17 is based on Debian Bullseye with just a JRE)
```

Comment:
I think this comment is out of date if you are switching the non-debug build to bookworm also.

Reply:
Yeah, better remove the name of a specific build of Debian from the comment, otherwise we will need to change it everytime the base image changes